IA/IA_upload.py

```diff
@@ -49,6 +53,31 @@ def mp_from_ids(mp_id: str, mp_keyname: str, bucket: str) -> MultiPartUpload:
     await asyncio.gather(*tasks)
 
 
+def upload_metadata(bucket_name: str, guid: str, directory: str):
+    session_data = {'s3': {'access': IA_ACCESS_KEY, 'secret': IA_SECRET_KEY}}
+    session = internetarchive.get_session(config=session_data)
+
+    node_path = os.path.join(HERE, directory, 'data', guid, 'node', '{}.json'.format(guid))
+    with open(node_path, 'r') as f:
+        node_json = json.loads(f.read())['attributes']
+
+    xml_path = os.path.join(HERE, directory, 'data', 'datacite.xml')
+    xml_data = minidom.parse(xml_path)
+    creators = xml_data.getElementsByTagName('creatorName')
+
+    item = session.get_item(bucket_name)
+    item.modify_metadata(dict(
+        title=node_json['title'],
+        description=node_json['description'],
+        creator=creators[0].firstChild.data,
+        date=node_json['date_created'],
```

Comment:
The date field should be only the year/month/day (eg, 2015-11-06) not a full timestamp (eg, 2015-11-06T18:12:24.416000).

IA docs: https://archive.org/services/docs/api/metadata-schema/index.html#date